src/control/lib/daos/ddb_types.go

```diff
@@ -0,0 +1,27 @@
+// (C) Copyright 2023 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+package daos
+
+import (
+	"github.com/google/uuid"
+)
+
+// CommandContext structure used for all commands
+type CommandContext struct {
+	DdbContext        interface{}
+	JsonOutput        bool
+	JsonOutputHandled bool
+}
```

Comment:
While I agree that `daos.Poolsuperblock` makes sense, I'm not sure that it makes sense to have the `CommandContext` here. Apologies if a prior comment lead you astray on this. The `lib/daos` package should be a home for C bindings to libdaos and other associated glue. This `CommandContext` type is only used by/for ddb, as far as I can tell. The fact that there is a `DdbContext` member which uses the empty interface (presumably to avoid circular import issues) is a strong signal that it doesn't belong here.

Reply:
Fixed